babel.cfg

```diff
@@ -0,0 +1,6 @@
+[python: src/olympia/**.py]
+[jinja2_custom: src/olympia/discovery/strings.jinja2]
+[django: src/olympia/**/templates/**/emails/**.*]
+[django: **/templates/admin/**.html]
+[django: **/templates/devhub/forms/widgets/compat_app_input_option.html]
```

Comment:
should these also start `/src/olympia`, at least so there's no attempt to parse files elsewhere?